packages/cadl-ranch-specs/http/content-type/multipart-formdata/mockapi.ts

```diff
@@ -0,0 +1,30 @@
+import { passOnSuccess, ScenarioMockApi, mockapi } from "@azure-tools/cadl-ranch-api";
+import { resolvePath } from "@typespec/compiler";
+import { readFileSync } from "fs";
+import { fileURLToPath } from "url";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+const root = resolvePath(fileURLToPath(import.meta.url), "../../../../../");
+
+const pngFile = readFileSync(resolvePath(root, "assets/image.png"));
+const jpgFile = readFileSync(resolvePath(root, "assets/image.jpg"));
+
+Scenarios.ContentType_MultipartFormData_multipart = passOnSuccess(
+  mockapi.post("/content-type/multipart-formdata", (req) => {
+    req.expect.deepEqual(req.body.id, "123");
```

Comment:
Anyway to verify the order of the parts? As it should be part of the spec (respect property order)

Reply:
I am not expert of multipart/form-data, but I don't find any declaration that users have to input property with specific order: https://swagger.io/docs/specification/describing-request-body/multipart-requests/.

Reply:
From those spec https://www.w3.org/TR/html4/interact/forms.html#h-17.13.4 about the html form -> formdata it does say it guarantees order from document order. Swagger might not care but TypeSpec spec does specify order of properties so I think it might be good to make sure parts are in order. But maybe lets discuss that in the multipart design meeting and keep this spec to the minimal example for now